### ISSUE TITLE
Contributors suck big time...

### ISSUE THREAD

**Issue by goodmanj2** [dotnet/docs#33283]
Is there no one who is not a show off, and can stuff their ego for a couple of minutes and show the most simple , most basic answer to a question?

Some of us out there are new to programming - and getting a basic handle on the programs would be of great help.

I am beginning to detest any programmer that supplies answers that are meant to show off how much they know.

There is a time and place for that - after the simple answer is given; then you can show off how much you know - that then would be helpful, as incremental learning...

I'm here because my college teachers also suffer from the same sickness...

This goes for ALL programming tutorials as well - basically, they suck...

Joel 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c7aa6134-64e2-28a7-f7d0-d0c1512f4520
* Version Independent ID: 3317fd4a-9cd3-57cd-66f7-abb271e302e8
* Content: [Strings - C# Programming Guide](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/)
* Content Source: [docs/csharp/programming-guide/strings/index.md](https://github.com/dotnet/docs/blob/main/docs/csharp/programming-guide/strings/index.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Hi @goodmanj2 

For beginner friendly content, I'd suggest starting [here](https://dotnet.microsoft.com/en-us/learn/csharp) or [here](https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world).

I can sense your frustration, and our team is working to improve the beginner experience. I'm going to close this issue because you haven't provided any details on what questions led you to this article, and what wasn't answered clearly.

We are interested in your feedback. That's one primary that helps us improve. That said, I recommend you read the [.NET Code of Conduct](https://dotnetfoundation.org/about/policies/code-of-conduct) and adhere to it when reaching out for help.

**Comment by goodmanj2** [comment]
code shmode -

Truly if you understood the frustration that is there for many beginning 
code students - your lessons would be much better - and, by the way, it 
is not just particular to you...

In one example I was trying to follow some code, and it seemed fairly 
clear, when suddenly a c# operator '$" appeared. I had seen it  before, 
but not in the context of its use in this code example.

I looked  and looked for an explanation and came across the same 
question in a code site. The writer had the same problem.

There it was - right in the code - and being an intelligent person - it 
was difficult to move on, whether or not the operator was absolutely 
necessary in this particular instance.

Code tutorials are filled with the appearance of unexplained symbols - 
just as one example.

  Code tutorials are really not clear enough

for example,  If statements could be demonstrated first as  simple CWL 
vars being compared and solved and then  using a CRL input comparing two 
variables - to keep the code example going.

In other words - to keep entering numbers (ie)  until  a certain 
criteria is met  - even / odd / larger than a certain threshold, or 
until a particular number is found, etc.

But, starting with the simple initial example and building on it 
develops an understanding.

And, there are real world examples, like requests for passwords, which 
can be a series of steps, passwords, number analysis etc.

Even as kids we used to play "Can I cross your bridge?"  (only if you 
were wearing a certain color.  So how is that done in code if you don't 
supply the color(s) in a CWL, but make the respondent supply an input 
and have that tested.

But, this is never done - nowhere.  The tutorials jump to another level, 
and introduce new concepts rather than build and build - learning a 
concept fully.

Some tutorials start with methods and arrays, others starts with 
computational thinking and flow charts. No one really starts at the 
beginning.

No one thinks in terms of learning to read - learning the alphabet, how 
to pronounce, how to construct basic sentences, and eventually learning 
more and more complicated syntax - considering different tenses, 
grammar, spelling and conjugations -  reading is pretty complicated - 
but it gets done - and the language is learned and made usable.

Tutorials and teachers - don't get it in teaching computing -

I don't have the time here to teach you how to teach - but you really 
aren't cutting it.

Joel
